tbx/static_src/sass/components/_call-to-action.scss

```diff
@@ -0,0 +1,29 @@
+@use "config" as *;
+
+.call-to-action {
+    background-color: var(--color--streamfield-background);
+    padding: $spacer-small;
+
+    @include media-query(large) {
+        padding: $spacer-medium;
+    }
+
+    &__inner {
+        @include media-query(large) {
+            display: flex;
+            align-items: flex-end;
+            justify-content: space-between;
+            gap: 40px;
```

Comment:
nitpick: you could use a spacer var here (2 * $spacer-mini-plus) - but maybe that is just a bit overkill as it starts to mean less in terms of the spacing scheme.